unit/unit.go

```diff
@@ -105,3 +107,105 @@ type SourceUnits []*SourceUnit
 func (v SourceUnits) Len() int           { return len(v) }
 func (v SourceUnits) Less(i, j int) bool { return v[i].String() < v[j].String() }
 func (v SourceUnits) Swap(i, j int)      { v[i], v[j] = v[j], v[i] }
+
+// sourceUnit is the struct for JSON serialization used to preserve
+// backcompat with the old SourceUnit JSON serialization format
+type sourceUnit struct {
```

Comment:
We should be able to remove this soon after switching over. It is just needed in the interim for toolchains whose docker images are not switched over immediately to the new srclib. I will add the comment.